zaza/openstack/charm_tests/kerberos/tests.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2020 Canonical Ltd.
```

Comment:
This test creatively introduces a new approach, not used elsewhere in Zaza to-date:  spinning up a generic cs:ubuntu unit and using that unit to execute client commands.  This is necessary here because the system needs to have Kerberos packages installed and the host needs to be configured with kinit, etc.  I think this is a sound and interesting approach.  We may want to distill some of this into general-purpose code at a later time, then update this test.